crowdfunding/users/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser #added 17/1
+
+class CustomUser(AbstractUser):
+    bio = models.CharField(blank=True, null=True, max_length=150)
```

Comment:
i would have considered a text field for the bio?
i guesss this is one way to keep it very short 😝